change/@ni-nimble-components-6aa0f191-427b-4745-9425-f602170a1266.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add spec for nimble-card, no code changes yet",
```

nit: Since the "type" already is "none", you don't need to mention in the comment that there aren't any code changes.

```suggestion
  "comment": "Add spec for nimble-card",
```